lib/cli/provider/index.js

```diff
@@ -154,6 +159,9 @@ module.exports.handler = function(argv) {
   , filter: function(device) {
       return argv.serial.length === 0 || argv.serial.indexOf(device.id) !== -1
     }
+    , exclude: function(device) {
```

Comment:
I would like to see this combined with the filter function. So instead of only doing the current check, it would check what it already does, and then this one. Then we can remove the exclusion code in `lib/units/provider/index.js` and just rely on the existing filter to do its job.